ui-nextjs/app/globals.css

```
@@ -105,3 +105,25 @@ button {
   transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
   transition-duration: 150ms;
 }
+
+/* Policy Bills feed: stagger-fade entrance for BillFeedRow. */
+@keyframes billFadeIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "billFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)